cpp/src/plasma/CMakeLists.txt

```diff
@@ -88,6 +88,13 @@ if (ARROW_GPU)
   add_definitions(-DPLASMA_GPU)
 endif()
 
+if (ARROW_USE_GLOG)
+  SET(PLASMA_LINK_LIBS glog_static ${PLASMA_LINK_LIBS})
```

Comment:
Why is this statically-linked twice? Shouldn't the symbols be in libarrow.so, which is loaded by libplasma.so?